deploy/kubernetes/examples/demo-storageclass-config-secret.yaml

```diff
@@ -0,0 +1,20 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: demo-storageclass
+provisioner: block.csi.ibm.com
+volumeBindingMode: WaitForFirstConsumer  # Optional. Required for Topology Awareness.

```

Comment:
in the context of a config storageclass yaml, I think the `Optional` here is still TBD, since current replication design doesn't allow a storageclass with a config secret (without topology)

Reply:
so what are you suggesting?

Reply:
omit the `Optional` here, at least until (if) replication design changes in a way that would indeed make this optional

Reply:
ok

Reply:
done